extensions/amp-subscriptions/0.1/viewer-subscription-platform.js

```diff
@@ -119,6 +119,7 @@ export class ViewerSubscriptionPlatform {
       .then(entitlementData => {
         entitlementData = entitlementData || {};
 
+        /** @deprecated Send error at top level of postMessage instead. */
```

Comment:
TBH, I'm not familiar with this form of deprecation. You can also just name this variable `deprecatedError` and keep the comment as is. I suppose this all just to make it clear to yourself that this is to be used sparingly now? You can also file an error on your WG and references it here.

Reply:
Good call, I updated the var name and comment